requirements.txt

```diff
@@ -2,5 +2,5 @@ django>=2.2,<3.1
 django-leaflet>=0.23,<0.27
 channels>=2.1.3,<2.5.0
 Pillow>=6.2.0,<=7.1.0
-openwisp-utils>=0.4.0,<0.5.0
```

Comment:
The error shown by `TravisBuddy` was due to the fact that `openwisp-utils version 0.4.0` was being  installed which doesn't have `openwisp-qa-checks`. 
This is the reason why i had to update it